src/Components/EmailPreview/EmailPreview.css

```diff
@@ -0,0 +1,11 @@
+.card-title {
+    font-size: 1.5rem;
+}
+
+.card-subtitle {
+    font-size: 1.3rem;
+}
+
+.card-text {
+    font-size: 1.2rem;
+}
```

Comment:
i think all of this can be done with bootstrap classes, look here:
https://getbootstrap.com/docs/5.3/utilities/text/#font-size

Reply:
deleted the css file as well